### PR TITLE
send email request

### PR DESCRIPTION
**Pull request by ghost**
email
## Related Tickets

- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/51481)

## WHAT (optional)

- Change number items `completed/total` in admin page.

## HOW

- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)

- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

### CODE REVIEW

**Comment by ghost** [comment]
ready

**Comment by ghost** [comment]
ready

File: app/controllers/admin/tour_requests_controller.rb [hunk 1]
```diff
@@ -27,6 +27,7 @@ def update
     respond_to do |format|
       format.js
     end
+    @tour_request.send_tour_request_email
```
Comment (manhnt-0960) [inline app/controllers/admin/tour_requests_controller.rb]: cho vào callback đi e

File: app/views/tour_request_mailer/auth_tour_request.html.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<h1><%= t ".title" %></h1>
+<p><%= t ".hi" %> <%= @tour_request.user.name %>,</p>
+<p>
+  <%= t ".content" %>
+</p>
+<p>
+  $<%= @tour_request.tour.price %><small> <%= t ".night" %></small></p>
+<h3><a><%= @tour_request.tour.name %></a></h3>
+<span><%= @tour_request.tour.category.name %></span>
```
Comment (manhnt-0960) [inline app/views/tour_request_mailer/auth_tour_request.html.erb]: dùng delegate đi e

**Comment by ghost** [comment]
ready

**Pull request merged**